go/private/actions/link.bzl

```
@@ -96,16 +96,18 @@ def emit_link(
         tool_args.add("-msan")
     if ((go.mode.static and not go.mode.pure) or
         go.mode.link != LINKMODE_NORMAL or
-        go.mode.goos == "windows" and (go.mode.race or go.mode.msan)):
+        go.mode.race or
+        go.mode.goos == "windows" and go.mode.msan):
```

```suggestion
        (go.mode.goos == "windows" and go.mode.msan)):
```
Makes precedence more obvious